hadoop-ozone/dist/src/shell/ozone/ozone

```diff
@@ -31,6 +31,7 @@ function hadoop_usage
   hadoop_add_option "--hosts filename" "list of hosts to use in worker mode"
   hadoop_add_option "--loglevel level" "set the log4j level for this command"
   hadoop_add_option "--workers" "turn on worker mode"
+  hadoop_add_option "--jvmargs" "jvm arguments"
```

Comment:
Could we add a bit more help text here? I'd like to make it clear the arguments here are appended to HADOOP_OPTS and if there are also any HADOOP_CLIENT_OPTS they are appended after. eg:

```
hadoop_add_option "--jvmargs" "jvm arguments" "append JVM options to any existing options defined in the HADOOP_OPTS environment variable. Any defined in HADOOP_CLIENT_OPTS will be append after these jvmargs"
```